src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/Message Center/MessagePage.cs

```diff
@@ -0,0 +1,153 @@
+﻿/*
+ Copyright Airship and Contributors
+*/
+
+using System;
+
+using Xamarin.Forms;
+
+namespace UrbanAirship.NETStandard.MessageCenter
+{
+    /// <summary>
+    /// ContentPage used for displaying Message Center Messages.
+    /// </summary>
+    public partial class MessagePage : ContentPage
+    {
+        /// <summary>
+        /// Gets and sets the message ID.
+        /// </summary>
+        /// <value>The message ID.</value>
+        public string MessageId { get; set; }
+
+        /// <summary>
+        /// Event handler invoked when the message has loaded.
+        /// </summary>
+        public event EventHandler<MessageLoadedEventArgs> Loaded;
```

Comment:
You might want to expose LoadStarted to make using the MessagePage easier, technically they should know when we trigger a reload, but this way they can sign up a loading indicator easily.

How does one handle a retry? Setting the messageId again?

Reply:
Yeah LoadStarted is a good call.

I didn't try reloading, but I think setting it again should be a reasonable way to go. We'll want to make sure it doesn't de-dupe though.

Reply:
Dang, that doesn't work. Might need to add a call to broadcast the change to the message ID property once the page has already loaded.

Reply:
Added property binding, as well as a LoadStarted event.